Dockerfile

```diff
@@ -1,24 +1,15 @@
-FROM  node:alpine
+FROM  node:alpine as builder
 
 # needed to build (python needed for compiling node-sass, for instance...)
 RUN apk add --no-cache \
     make \
     gcc \
     g++ \
-    less \
-    vim \
-    psmisc \
-    net-tools \
     git \
-    tig \
-    wget \
-    curl \
-    openssh-client \
-    python \
-    bash 
+    python
 
-RUN mkdir /opt/pizza-sync
-COPY . /opt/pizza-sync/
+WORKDIR /opt/pizza-sync
+ADD . .
 
 # frontend
 RUN cd /opt/pizza-sync/frontend && yarn && yarn run build:prod && rm -rf node_modules
```

Comment:
no need to remove node_modules, it stays in the build image :)

Reply:
Hmm, ok, but what to do with this build image that takes disk space ?

Reply:
does it? it's not very important, is it? but ok, why not, you decide :)